docs/source/using/addons.adoc

```diff
@@ -75,6 +77,18 @@ This can be useful in cases where you know that a command such as `oc` might tak
 
 NOTE: Trying to use an undefined command will cause an error when the add-on gets parsed.
 
+[[addon-openshift-version-semantics]]
+== OpenShift Version Semantics
+
+Minishift allows the use of OpenShift version within add-on metadata.
+It use below semantic and compare with running OpenShift cluster in Minishift.
+
+----
+- v3.6.0+     : Add-on can run on OpenShift version 3.6.0 or higher
```

Comment:
OpenShift version 3.6.0 or higher needs to be running in order to apply the add-on.

Reply:
Why '+'? On what are you basing this syntax? Is when it comes to version ranges not something like > and >= more common? It is also less confusing when thinking about our actual Minishift versions. We have something like 'minishift v1.6.0+2b005fb', meaning we use the '+' to separate the sha part. I initially thought the '+' was some sort of left over from you cutting of the sha.

Reply:
What's about things like 3 or 3.6? Is this supported? Implementation wise I think it might be.